zha/units.py

```diff
@@ -148,6 +148,17 @@ class UnitOfConductivity(StrEnum):
     MILLISIEMENS_PER_CM = "mS/cm"
 
 
+class UnitOfSpeed(StrEnum):
```

Comment:
Let's use the definitions from Home Assistant, to be complete:

```python
class UnitOfSpeed(StrEnum):
    """Speed units."""

    BEAUFORT = "Beaufort"
    FEET_PER_SECOND = "ft/s"
    INCHES_PER_SECOND = "in/s"
    METERS_PER_SECOND = "m/s"
    KILOMETERS_PER_HOUR = "km/h"
    KNOTS = "kn"
    MILES_PER_HOUR = "mph"
    MILLIMETERS_PER_SECOND = "mm/s"
```

Reply:
done